dev-packages/electron/electron-replace-ffmpeg.js

```diff
@@ -60,7 +60,7 @@ async function main() {
 /**
  * @typedef {Object} File
  * @property {String} name
- * @property {String|undefined} folder
+ * @property {String} [folder]
```

Comment:
I have linter errors in this module:
```
Type '{ name: string; }' is not assignable to type 'File'.
Property 'folder' is missing in type '{ name: string; }'
.ts(2322)
```

Since we define the API, can we get rid of the issues, please?

Reply:
I was on the `master`, sorry. It works as expected on `mp/electron-no-h264`.